src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -634,6 +634,18 @@ void assertPhrasingContent() throws InvalidInputException {
         LineBreak.class, Span.class, Emoji.class, HashTag.class, CashTag.class, Mention.class));
   }
 
+  /**
+   * Check that the element's children are limited to allowed element types returning a specific message given in input.
+   */
+  void assertContentModel(Collection<Class<? extends Element>> permittedChildren, String message)
+      throws InvalidInputException {
+    try {
+      assertContentModel(permittedChildren);
+    } catch (InvalidInputException e) {
+      throw new InvalidInputException(message);
```

Comment:
do we hide the original exception on purpose here? vs chaining it with throw new InvalidInputException(message, e) ?

Reply:
I wanted to make it more specific that's why I'm completely replacing the original message. 
It was too generic for this usecase.